src/MessageRouter.py

```diff
@@ -0,0 +1,47 @@
+import time
+import re
+
+
+class MessageRouter:
+    """
+    Responsible for invoking a message command handler if a message contains
+    a command and the rate limit is not exceeded.
+    """
+
+    def __init__(self,
+                 command_handlers={},
+                 default_rate_limit_in_seconds=60,
+                 per_command_rate_limit_in_seconds={}):
+        self._default_rate_limit = default_rate_limit_in_seconds
+        self._command_handlers = command_handlers
+        self._per_command_rate_limit = per_command_rate_limit_in_seconds
+
+        self._time_of_last_command_usage = {}
+
+    def handle_message(self, message):
```

Comment:
What if a command handler needs to reply to a different channel than `#aeolus` or perform other IRC communication?